frontend/packages/dev-console/integration-tests/features/serviceMesh/add-Namespace-ToServiceMesh-Admin.feature

```diff
@@ -0,0 +1,82 @@
+@serviceMesh
+Feature: Associate Kiali Dashboard to Topology filterbar by adding a namespace to the Service Mesh by Admin user
+    User should be able to add a namespace to Service Mesh to get the Kiali dashboard link for that namespace on the Topology filterbar
+
+
+    Background: 
+        Given user has logged in as kubeadmin
+        And user is at Administrator perspective
+        And user has installed ElasticSearch Operator provided by Red Hat
+        And user has installed Red Hat OpenShift Jaeger Operator
+        And user has installed Kiali Operator provided by Red Hat
+        And user has installed Red Hat OpenShift Service Mesh Operator
+        And user has created new project "istio-system"
+        And user has created Istio Service Mesh Control Plane instance in "istio-system" namespace
+        And user has created Istio Service Mesh Member Roll instance
```

Comment:
just to be double sure , @vikram-raj  does above steps looks good ?

Reply:
Commited the suggested change by @vikram-raj

Reply:
@makambalaji I am not seeing the changes here suggested by me.